google-cloud-examples/src/main/java/com/google/cloud/examples/bigtable/HelloWorld.java

```diff
@@ -0,0 +1,167 @@
+
+public class HelloWorld {
+
+  private static final String COLUMN_FAMILY = "cf1";
+  private static final String COLUMN_QUALIFIER = "greeting";
+  private static final String ROW_KEY_PREFIX = "rowKey";
+  private final String tableId;
+  private final BigtableDataClient dataClient;
+  private final BigtableTableAdminClient adminClient;
+
+  public static void main(String[] args) throws Exception {
+
+    if (args.length != 2) {
+      System.out.println("Missing required project id or instance id");
+    }
+    String projectId = args[0];
+    String instanceId = args[1];
+
+    HelloWorld helloWorld = new HelloWorld(projectId, instanceId, "test-table");
+    helloWorld.run();
+  }
+
+  public HelloWorld(String projectId, String instanceId, String tableId) throws IOException {
+    this.tableId = tableId;
+
+    // [START connecting_to_bigtable]
+    // Create the settings to configure a bigtable data client
+    BigtableDataSettings settings =
+        BigtableDataSettings.newBuilder()
+            .setInstanceName(InstanceName.of(projectId, instanceId))
+            .build();
+
+    // Create bigtable data client
+    dataClient = BigtableDataClient.create(settings);
+
+    // Create the settings to configure a bigtable admin client
+    BigtableTableAdminSettings adminSettings =
+        BigtableTableAdminSettings.newBuilder()
+            .setInstanceName(com.google.bigtable.admin.v2.InstanceName.of(projectId, instanceId))
+            .build();
+
+    // Create bigtable admin client
+    adminClient = BigtableTableAdminClient.create(adminSettings);
+    // [END connecting_to_bigtable]
+  }
+
+  public void run() throws Exception {
+    createTable();
+    writeToTable();
+    readSingleRow();
+    readTable();
+    deleteTable();
+
+    dataClient.close();
+    adminClient.close();
+  }
+
+  public void createTable() {
+    // [START creating_a_table]
+    // Check if table exists, create table if does not exist
+    if (!adminClient.exists(tableId)) {
+      System.out.println("Creating table: " + tableId);
+      CreateTableRequest createTableRequest =
+          CreateTableRequest.of(tableId).addFamily(COLUMN_FAMILY);
+      adminClient.createTable(createTableRequest);
+      System.out.printf("Table %s created successfully%n", tableId);
+    }
+    // [END creating_a_table]
+  }
+
+  public void writeToTable() {
+    // [START writing_rows]
+    try {
+      System.out.println("\nWriting some greetings to the table");
+      String[] greetings = {"Hello World!", "Hello Bigtable!", "Hello Java!"};
+      for (int i = 0; i < greetings.length; i++) {
+        RowMutation rowMutation =
+            RowMutation.create(tableId, ROW_KEY_PREFIX + i)
+                .setCell(COLUMN_FAMILY, COLUMN_QUALIFIER, greetings[i]);
+        dataClient.mutateRow(rowMutation);
+        System.out.println(greetings[i]);
+      }
+    } catch (NotFoundException e) {
+      System.err.println("Exception while writing to table: " + e.getMessage());
```

Comment:
Please be more specific:
System.err.println("Failed to write to a non-existent table");